lib/API/JSON/v1/Services.php

```diff
@@ -85,6 +101,13 @@ public function checkMSSKey($data = []) {
       ]);
     }
 
+    // pause sending when key is pending approval, resume when not pending anymore
+    if ($this->servicesChecker->isMailPoetAPIKeyPendingApproval()) {
```

Comment:
I think we can avoid pausing sending multiple times by only pausing if the key was not pending approval before. What do you think?